packages/address-book/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@yieldhubfinance/blockchain-addressbook",
-  "version": "0.4.0",
+  "version": "0.5.0",
```

Comment:
We should decide on a versioning that isn't a major bump for every change, but LGTM for now

Reply:
ah yeah.. i thought this was already minor version since i thought first one would be major but you're right